admin.py

```diff
@@ -1,8 +1,25 @@
 from django.contrib import admin
 
-# Register your models here.
```

Comment:
This should stay in there for future contributions

Reply:
Fixed.